### ISSUE TITLE
Do you think is possible to Mimic the Advance Voice com Openai?

### ISSUE THREAD

**Issue by rubenssoto** [PromtEngineer/Verbi#13]
I would like to know if it is possible to mimic the openai advanced voice, I would like to test but I dont have a clue how to do that.

**Comment by SmartManoj** [comment]
Yes. [Setup Instructions](https://github.com/PromtEngineer/Verbi#setup-instructions--)
